app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ActiveRecord::Base
+end
```

Comment:
Please add validations to this model. We should check that `username` and `token` are not `nil` (presence).